spring-boot-admin-server-ui/src/main/frontend/views/instances/scheduledtasks/scheduled-task-executions.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <td class="font-mono text-sm">
+    <sba-formatted-obj :value="task.nextExecution.time" />
+  </td>
+  <td class="font-mono text-sm">
+    <sba-formatted-obj
+      v-if="task.lastExecution"
+      :value="task.lastExecution.time"
+    />
+  </td>
+  <td class="font-mono text-sm">
+    <span
+      v-if="task.lastExecution"
+      role="status"
+      :class="
+        classNames('status-badge', task.lastExecution.status.toLowerCase())
+      "
```

Comment:
Since we don't need to conditionally join class names here, we could use simple string concatenation and get rid of the `classNames` dependency.
```suggestion
      :class="`status-badge ${task.lastExecution.status.toLowerCase()}`"
```